envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/debug/SnapshotDebugController.kt

```diff
@@ -61,6 +68,11 @@ class SnapshotDebugController(controlPlane: ControlPlane) {
         }
     }
 
+    @PostMapping("/chaos/fault/read_network_delay")
```

Comment:
how about `read-network-delay`, according to the convention we already have, e.g. `snapshot-global`

Reply:
Sure, no problem :)